src/frontend/screens/Settings/components/Tools/index.scss

```
@@ -20,7 +27,12 @@
     text-decoration: none;
     white-space: nowrap;
 
-    & > span {
+    & div {
+      position: relative;
+      top: -18px;
+    }
```

```suggestion
    justify-content: center;
```

(I left another comment to remove the div, and with this css it fixes the alingment)

---

fixed